test/trimWhitespaces.test.js

```diff
@@ -0,0 +1,15 @@
+const { trimWhitespaces } = require("../lib");
```

Comment:
I think this would be better referred to as `removeWhitespace`. Trim would imply just removing whitespace from the ends.

Reply:
Should be fixed now